src/components/Image/Image.js

```diff
@@ -33,15 +33,27 @@ const LazyLoadedImage = (props) => {
     // the low quality image (props.image.thumbnail) will be hidden
   };
 
+  // This block checks whether the user specified fallback image actually works or not
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValidFallback, setIsValidFallback] = useState(false);
```

Comment:
What's the reason for handling the error of the fallback image? The fallback image is likely to be a static image, which is quite stable. The user can handle it using their code if they want to, isn't it?

Reply:
We can simply assign the fallback image in the `handleError` function.

Reply:
This was intended to catch cases where a user provides an invalid/broken url for the fallback image - in that event, I wanted to make sure that the carousel would end up falling back onto the built-in placeholder again.